tests/whitebox/_misc.py

```diff
@@ -171,6 +171,11 @@ def setUp(self):
 
 RUNNER = run()
 
+class StratisCliTestRunError(Exception):
```

Comment:
Now make this a subtype of ```AssertionError```, that's the vital part that will the unittest framework to interpret the exception as indicating a test failure, not an error.